toc/working-groups/002-PRs-Require-2-Reviewers.md

```diff
@@ -0,0 +1,17 @@
+# PRs Require Two Reviewers
+
+Date of Forum to be Discussed:
+Author: Amelia Downs - @ameowlia
+
+## Problem
+We have more approvers now (a great problem to have)! Let's make sure we keep a transparent and high level
+of code quality.
+
+## Solution
+1. Each PR requires 2 points before it can be merged.
+1. Each approver or tech lead can give 1 point by commenting on the PR with
+   "+1".
```

Comment:
+1 in comments, or can we just use the built-in review/approve features of github? I